prow/external-plugins/cherrypicker/server.go

```diff
@@ -177,7 +194,7 @@ func (s *Server) handleIssueComment(l *logrus.Entry, ic github.IssueCommentEvent
 				return s.ghc.CreateComment(org, repo, num, plugins.FormatICResponse(ic.Comment, resp))
 			}
 		}
-		resp := fmt.Sprintf("once the present PR merges, I will cherry-pick it on top of %s in a new PR and assign it to you.", targetBranch)
+		resp := fmt.Sprintf("once the present PR merges, I will cherry-pick it on top of %s/%s#%s in a new PR and assign it to you.", org, repo, targetBranch)
```

Comment:
You need a reverse formatter:

```go
func formatTarget(org, repo, branch string) string {
	var target string
	if org != "" {
		target += org
		target += "/"
	}
	if repo != "" {
		target += repo
		target += "#"
	}
	target += branch
	return target
}
```

Reply:
My intention here is to print all the info (org/repo#branch) regardless the request. Even if the user requested to cherrypick only a branch, still those values won't be empty.